app/celery/common.py

```diff
@@ -27,14 +27,13 @@ def handle_max_retries_exceeded(notification_id: str, method_name: str) -> str:
     return message
 
 
-def handle_non_retryable(notification_id: str, method_name: str) -> str:
+def handle_non_retryable(notification_id: str, method_name: str):
```

Comment:
I'd suggest adding annotation ->None, which is rarely used for `__init__` , but should be used for other methods.

Reply:
Done